opendevin/runtime/plugins/agent_skills/agentskills.py

```diff
@@ -230,17 +235,17 @@ def _cur_file_header(current_file, total_lines) -> str:
 
 @update_pwd_decorator
 def open_file(
-    path: str, line_number: int | None = 1, context_lines: int | None = 100
+    path: str, line_number: int | None = 1, context_lines: int | None = WINDOW
```

Comment:
WINDOW should be 300?

Reply:
Good catch! It is actually 100 - i experimented earlier with 300 and it didn't go well so i reverted back to 100, but forgot to revert the doc back :(

Reply:
There are 2 more locations of 300, I think